examples/httpserver_simpletest_connectionmanager.py

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: 2024 DJDevon3
+# SPDX-License-Identifier: MIT
+# Coded for Circuit Python 9.
+"""HTTP Server Simpletest with Connection Manager"""
+# pylint: disable=import-error
+
+import os
+
+import adafruit_connection_manager
+import wifi
+
+from adafruit_httpserver import Server, Request, Response
+
+# Get WiFi details, ensure these are setup in settings.toml
+ssid = os.getenv("WIFI_SSID")
```

Comment:
@michalpokusa, which env names do you want for examples in this repo? Should it not use the `CIRCUITPY_*` ones so it doesn't auto connect?

Reply:
Only the `httpserver_simpletest_manual.py` example uses env names at runtime, other examples utilize `settings.toml` with `CIRCUITPY_*` envs and auto connect feature.

Either way, if something is not consistent with other examples I will correct it in my PR, e.g. missing emphasised lines or description of example in docs.

Reply:
I would probably use `CIRCUITPY_*`, where possible, since people playing with different things would most likely already have those on their devices...

Reply:
They are only used in one example that clearly states that this is a manual approach. Every other example uses `settings.toml`, it is just to show there are multiple possibilities.

![image](https://github.com/adafruit/Adafruit_CircuitPython_HTTPServer/assets/72110769/6d805db7-f0ad-4d84-9ab1-8db185f3dd7c)

If I am correct, setting the `CIRCUITPY_*` envs also turns on the web workflow, which might not be desired.

Reply:
Ahhh. Yeah and that was my thought, if the general ones are used, it auto connects, and thus it can be used for web workflow

Reply:
So I did the right thing by setting it back to how the manual example has it?

Reply:
I am not sure how I will order the examples. IMO right now it can stay like that and if it becomes necessary I will change that in my PR.

Reply:
Sounds good to me 👍